File: Planification/src/entities/Film.java
package entities;

public class Film
{
    private int id;
    private String titre;
    private int duree;
    private String pays;
    private int idRealisateur;
    
    public Film(int id, String titre, int duree, String pays, int idRealisateur)// Permet de stocker les informations sur un film
    {
        this.id = id;
        this.titre = titre;
        this.duree = duree;
        this.pays = pays;
        this.idRealisateur = idRealisateur;
    }

    public int getId()
    { return id; }

    public String getTitre()
    { return titre; }

    public int getDuree()
    { return duree; }

    public String getPays()
    { return pays; }

    public int getIdRealisateur()
    { return idRealisateur; }
}

File: Hebergement/entities/Film.php
<?php

class Film
{
  private $_id = null;
  private $_titre = null;
  private $_duree = null;
  private $_pays = null;
  private $_idRealisateur = null;

  public function __construct($id, $titre, $duree, $pays, $idRealisateur)
  {
    $this->_id = $id;
    $this->_titre = $titre;
    $this->_duree = $duree;
    $this->_pays = $pays;
    $this->_idRealisateur = $idRealisateur;
  }

  public function getId()
  { return $this->_id; }

  public function getTitre()
  { return $this->_titre; }

  public function getDuree()
  { return $this->_duree; }

  public function getPays()
  { return $this->_pays; }

  public function getIdRealisateur()
  { return $this->_idRealisateur; }
}
 ?>
